<launch>
    <arg name="robot_ns" default="robot0" />
    <node name="robot_no_crash_node" type="robot_no_crash" pkg="robot_no_crash" ns="$(arg robot_ns)" output="screen" />
    <param name="wall_dist" value="0.5" type="double" />
    <remap from="laser_1" to="laser_0" />
</launch>
